### PR TITLE
Enable strict null checking for Notifications test

### PR DESCRIPTION
**Pull request by usernamehw**
#65233 

- [x] `"./vs/workbench/test/common/notifications.test.ts"`

### CODE REVIEW

**Comment by usernamehw** [comment]
I can change ` as any as string` to `!` if the style should be consistent. I just didn't know it's possible to write something like this `null!`...

**Comment by mjbvz** [comment]
Thanks!

File: src/vs/workbench/test/common/notifications.test.ts [hunk 1]
```diff
@@ -15,14 +15,14 @@ suite('Notifications', () => {
 
 		// Invalid
 		assert.ok(!NotificationViewItem.create({ severity: Severity.Error, message: '' }));
-		assert.ok(!NotificationViewItem.create({ severity: Severity.Error, message: null }));
+		assert.ok(!NotificationViewItem.create({ severity: Severity.Error, message: null! }));
```
Comment (epicfaace) [inline src/vs/workbench/test/common/notifications.test.ts]: @usernamehw what does this mean? Are you asserting that `null` is not null?

Comment (epicfaace) [reply src/vs/workbench/test/common/notifications.test.ts]: Specifically, what does `null!` mean? I thought that would mean to the compiler that `null` cannot be null, which doesn't make sense to me.

Comment (usernamehw) [reply src/vs/workbench/test/common/notifications.test.ts]: I learned it yesterday myself. Yes, that's exactly what it means.

Those tests are bound to be failed. null is strict checked and cannot be passed as message, but there is a need to have a failing tests without thousands of compile errors in project. (And preferably by not casting everything to `any` to not loose autocomplete).

**Pull request merged**